Added badges and improved tracking

txrp0x9 opened the pull request:
## About
This PR aims to add badge functionality and better contributor tracking.
Resolves #14 
## Changes
### Badge
- Added two types of badges, a **predefined badge** and a **special badge**.
  - A **Predefined Badge** is handed out to every contributor depicting their progress with the org. Examples include a "Baby Steps" badge, "Beginner" badge, "Intermediate" badge, "Expert" badge, and so on.
  - A **Special Badge** is only awarded to those who have special accomplishments which differentiates them from others. Examples include, "Top Contributor of all time", "Top Contributor of the week", "Top Contributor of the month", "Top Contributor of the year", and so on.
- Also added tooltips to make it look shiny
![ss1](https://user-images.githubusercontent.com/41499871/71488566-3f6b3100-2847-11ea-8576-d350edd2b0cb.png)

- Added a `/badge` route which contains all information of the predefined and special badges.
![ss2](https://user-images.githubusercontent.com/41499871/71488573-45611200-2847-11ea-818d-dcc344545909.png)

### Tracking
Added a method to sort all top contributors of a repo/org and have them displayed below the repo/org.

### Other small changes
- Added a button that allows users to add more organizations to their list even after they have logged in.
_________________________________________

ayushnagar123 commented:
@shivamluthra not sure why organization with many commits taking an infinite time to load. Need to improve this. Any idea @Torpedo99 .

Torpedo99 commented:
> @shivamluthra not sure why organization with many commits taking an infinite time to load. Need to improve this. Any idea @Torpedo99 .

Fixed this bug in the above commit

txrp0x9 committed:
Added Badges and Contributor Tracking

Improved contributor tracking

Added Predefined Badges

Added Special Badges

Added a /badge route and tooltips

shivamluthra reviewed (approved):
Great work as expected! :+1: @Torpedo99
